docs/advisories/etcd-manager-certificate-expiration.md

```diff
@@ -0,0 +1,64 @@
+# etcd-manager Certificate Expiration
+
+etcd-manager configures certificates for TLS communication between kube-apiserver and etcd, as well as between etcd members.
+These certificates are signed by the cluster CA and are valid for a duration of 1 year.
+
+Affected versions of etcd-manager currently do NOT automatically rotate these certificates before expiration.
+If these certificates are not rotated prior to their expiration, Kubernetes apiserver will become inaccessible and your control-plane will experience downtime.
+
+## How do I know if I'm affected?
+
+Clusters are affected by this issue if they're using a version of etcd-manager < `3.0.20200428`.
+The etcd-manager version is set automatically based on the Kops version.
+These Kops versions are affected:
+
+* Kops 1.10.0-alpha.1 through 1.16.1
+* Kops 1.17.0-alpha.1 through 1.17.0-beta.1
+* Kops 1.18.0-alpha.1 through 1.18.0-alpha.2
+
+The issue can be confirmed by checking for the existence of etcd-manager pods and observing their image tags:
+
+`kubectl get pods -n kube-system -l k8s-app=etcd-manager-main -o jsonpath='{..image}' | tr -s '[[:space:]]' '\n'`
+
```

Comment:
This should produce a slightly cleaner output. Also making code blocks as "bash" may help with formatting.
````suggestion

```bash
kubectl get pods -n kube-system -l k8s-app=etcd-manager-main \
  -o jsonpath='{range .items[*].spec.containers[*]}{.image}{"\n"}{end}'
```

````

Reply:
good idea 👍  I added that. Ideally we could get pods for both main and events clusters but there isn't a label that matches both sets of pods.

Reply:
I don't think there is a case where they run different versions of `etcd-manager`. Best not tu confuse people.

Reply:
Each etcdCluster has its own Image field that could theoretically be overridden separately. It's probably not worth mentioning in this section but I am considering mentioning that in the 2nd Solution instructions.